package.json

```
@@ -54,7 +54,7 @@
 				"title": "Select active configuration"
 			},
 			{
-				"command": "mirrord.submitFeedback",
+				"command": "mirrord.joinDiscord",
 				"category": "mirrord",
 				"title": "Submit feedback"
```

Should also be updated, no?

maybe
```suggestion
				"title": "Join our Discord"
```

---

Or 
```suggestion
				"title": "Get help on Discord"
```